doc/user_guide/docs/visualization.md

```diff
@@ -55,6 +55,15 @@ In the next window keep the default (_Use File Names From State_) and click OK.
 
 ![ParaView Load State Data](images/pv_load_state_3.png)
 
+One can also directlly access paraview via utilising a pipeline from the terminal :
+
+```
+make - j8 && ./project && paraview output/project/*.pvsm
```

Comment:
Could you please add the info that this commands must be executed from within the `build` directory?